backend/init_data/skills/subagent/create_subagent_task.py

```
@@ -0,0 +1,750 @@
+
+Note: This operation may take several minutes for complex tasks."""
+
+    args_schema: type[BaseModel] = CreateSubAgentTaskInput
+
+    # Injected dependencies - set when creating the tool instance
+    task_id: int = 0
+    subtask_id: int = 0
+    ws_emitter: Any = None
+    user_id: int = 0
+    user_name: str = ""
+    bot_config: list = []  # Bot config list [{shell_type, agent_config}, ...]
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)